### DISCUSSION TITLE
Is it possible to write plugins adding new rules?

### DISCUSSION THREAD

**Discussion by edumudu** [biomejs/biome#863] [Q&A] [answered]
In ESLint, you can write a plugin to add new rules, which can be very useful. However, I cannot find in the documentation a way to achieve the same functionality with Biome. Is there a method available, or is it planned to be implemented in the future?

**Comment by ematipico** [comment, marked as answer]
It is planned, but we can't give more information at the moment. We still need to evaluate how to provide such functionality

**Reply by JosXa** [reply]
We use ESLint together with Biome now. It's a bit of a hassle, but not being able to migrate our important custom ESLint rules has been a bit of a blocker so far. Is there a roadmap and ETA?

**Reply by arendjr** [reply]
There is no ETA, but plugin support is being tracked here: https://github.com/biomejs/biome/issues/2463
